osschallenge/tests/pages/project.py

```diff
@@ -0,0 +1,15 @@
+class ProjectPage(object):
+
+    def __init__(self, selenium_driver, live_server_url):
+        self.driver = selenium_driver
+        self.live_server_url = live_server_url
+
+    def open_page_one_projects(self, page):
+        self.driver.get(
+            "{}/projects/{}/".format(self.live_server_url, page)
+        )
+        return self
+
+    def find_active_page(self):
+        active_page = self.driver.find_element_by_xpath("//a[@name='active']")
```

Comment:
Can't you also use a CSS selector for this? `a[name="active"]` or something?

Reply:
Yes that works too. Is it always better to use CSS selector instead of xpath?

Reply:
I'd say so, xpath seems quite complicated...